### PR TITLE
openapi jaxb 2.0

### LINKED ISSUE #18: jaxb 4.0 upgrade
jaxb 4.0 support

### PR DESCRIPTION
**Pull request by petrenyi-mark**
closes #18 

* remove swagger-jaxb dependencies
* refactor processing to primary work with Outlines
* jaxb 2.3 -> 4.0
* microprofile-openapi 1.0 -> 2.0
* replace demo project with unit tests
* introduce icellmobilsoft oss parent pom
* add default github actions

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -3,364 +3,19 @@
...
-[source,xml]
 ----
-<xs:annotation>
-    <xs:documentation xml:lang="hu">szöveg</xs:documentation>
-    <xs:documentation xml:lang="en">text</xs:documentation>
-</xs:annotation>
+    <dependency>
+        <groupId>hu.icellmobilsoft.jaxb</groupId>
+        <artifactId>openapi-jaxb-plugin</artifactId>
+        <version>2.0.0</version>
```
Comment (rombow) [inline README.adoc]: Ne egessuk bele a verzioszamot, kifelejtjuk frissiteni

File: docs/index.adoc [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+openapi-jaxb plugin
+
+== Maven central
+This plugin is available in maven central as:
+[source, xml]
+----
+    <dependency>
+        <groupId>hu.icellmobilsoft.jaxb</groupId>
+        <artifactId>openapi-jaxb-plugin</artifactId>
+        <version>2.0.0</version>
```
Comment (rombow) [inline docs/index.adoc]: itt is valami utalast csak a verzioszamra kellene

File: openapi-jaxb-plugin/pom.xml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        <artifactId>openapi-jaxb</artifactId>
+        <version>2.0.0-SNAPSHOT</version>
+    </parent>
+
+    <groupId>hu.icellmobilsoft.jaxb</groupId>
+    <artifactId>openapi-jaxb-plugin</artifactId>
+
+    <properties>
+        <jaxb.impl.version>4.0.1</jaxb.impl.version>
+        <microprofile-openapi-api.version>2.0.1</microprofile-openapi-api.version>
```
Comment (rombow) [inline openapi-jaxb-plugin/pom.xml]: Doksiban 3.0.0 van irva

**Review by rombow — DISMISSED** [review]

File: .github/workflows/compile_maven.yml [hunk 4]
```diff
@@ -0,0 +1,41 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Compile with Maven
```
Comment (speter555) [inline .github/workflows/compile_maven.yml]: https://github.com/i-Cell-Mobilsoft-Open-Source/parent-pom/blob/master/.github/workflows/compile.yml copy paste lehetne

File: .github/workflows/docs.yml [hunk 5]
```diff
@@ -0,0 +1,43 @@
+# This workflow will generate html documentation from asciidoc and publish it to `gh-pages` branch
+name: Deploy documentation
+on:
+  push:
+    branches:
+      - master
+
+jobs:
```
Comment (speter555) [inline .github/workflows/docs.yml:7]: copy paste https://github.com/i-Cell-Mobilsoft-Open-Source/parent-pom/blob/master/.github/workflows/docs.yml

**Review by rombow — DISMISSED** [review]

**Review by speter555 — DISMISSED** [review]

**Review by bucherarnold — DISMISSED** [review]

**Review by czenczl — DISMISSED** [review]

**Review by czenczl — DISMISSED** [review]

**Review by speter555 — DISMISSED** [review]

**Review by bucherarnold — DISMISSED** [review]

File: .github/workflows/manual_release.yml [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+          key: ${{ secrets.SSH_PRIVATE_KEY_GITHUB_TECHUSER }}
+          name: id_rsa # optional
+          known_hosts: ${{ secrets.KNOWN_HOSTS }} # jelenleg ures a tartalma
+          # config: ${{ secrets.CONFIG }} # ssh_config; optional
+
+      - name: Checkout ${{ github.event.ref }}
+        uses: actions/checkout@v2
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
```
Comment (rombow) [inline .github/workflows/manual_release.yml]: ez tuti regi

**Review by rombow — DISMISSED** [review]

File: .github/workflows/manual_release.yml (hunk 6 above)
Comment (petrenyi-mark) [reply .github/workflows/manual_release.yml]: jogos ez kimaradt - frissítettem

Comment (rombow) [reply .github/workflows/manual_release.yml]: meg mindig a regi workflow latszodik

**Review by rombow — CHANGES_REQUESTED** [review]

**Review by rombow — APPROVED** [review]

**Review by czenczl — APPROVED** [review]

**Review by speter555 — APPROVED** [review]

**Review by bucherarnold — APPROVED** [review]

**Pull request merged**